### DISCUSSION TITLE
Jump on an ID of a function placed under commented code

### DISCUSSION THREAD

**Discussion by JMGrenhas** [microsoft/pylance-release#2644] [Enhancement]
Hi!
It would be great that Pylance permitted to jump, on an ID of a function placed under comments, to the respective definition, as it does at normal code. Thanks.
Ex.:
'''
My text here mentions myFunction().
or...
'''
\# My text here mentions myFunction().
=> Ctrl+click on the ID myFunction() would jump to def(inition) of the function.

**Comment by judej** [comment]
Thanks for the suggestion @JMGrenhas!

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Reply by JMGrenhas** [reply]
Thanks. It would be very usefull when referencing symbols (functions, etc.) on comments.
One more thing: inside comments, everytime you type a dot (".") the editor pops up a menu with symbols of no interest.
